lte/gateway/c/core/oai/include/map.h

```diff
@@ -218,6 +218,40 @@ struct map_s {
       return MAP_KEY_NOT_EXISTS;
     }
   }
+
+  /***************************************************************************
+  **                                                                        **
+  ** Name:    isEmpty()                                                     **
+  **                                                                        **
+  ** Description: Returns true if map is empty, else returns false           **
+  **                                                                        **
+  ***************************************************************************/
+  bool isEmpty() {
+    if (umap.empty()) {
```

Comment:
Please rebase as this has changed on master

Reply:
Addressed review comment